kroxylicious-api/src/main/java/io/kroxylicious/proxy/config/tls/TrustProvider.java

```diff
@@ -28,4 +28,7 @@ public interface TrustProvider {
      */
     <T> T accept(TrustProviderVisitor<T> visitor);
 
+    default String hostnameVerificationAlgorithm() {
```

Comment:
Alters the public API in a minimally invasive way but still leaves the defaulting to happen in two places.